week-3/refactor/format-as-12-hours.js

```diff
@@ -4,3 +4,55 @@
 // Store this expression in a variable and reference it twice in the function in the correct place
 
 // Explain why it makes more sense to store this expression in a variable
+
+function formatAs12HourClock(time) {
+    if (Number(time.slice(0, 2)) > 12) {
```

Comment:
this is a bit complicated, can you think of a simpler solution where you can get the hours, and check if its less than 12 or not, to decide the `am` or `pm` and also use that to get the right time

Reply:
okay, thank you for checking my work.